modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceDiscounts.testcase

```diff
@@ -1103,4 +1103,44 @@ definition {
 			locator1 = "CommerceEntry#WORKFLOW_STATUS");
 	}
 
+	@description = "This is a test for COMMERCE-10663. When a single approver is set workflow is triggered once the user clicks on 'Submit for Publication"
+	@priority = "3"
+	test ViewWorkFlowIsCorrectAppliedWhenSingleApproverIsSet {
+		property portal.acceptance = "false";
+
+		task ("Given Single Approver is set") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
+
+			Navigator.gotoNavItem(navItem = "Configuration");
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "Commerce Discount");
+		}
+
+		task ("And new discount is added and discount is submitted for publication") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Discounts");
+
+			Click(locator1 = "Button#PLUS");
+
+			CommerceEntry.addNewDiscountsEntry(
```

Comment:
@Brunno0liveira In this way we are not checking the "Draft" status before hitting "Submit for Publications", yet only the final one. I would suggest one between the following solutions:

1. Adjusting the macro by inserting an optional variable (e.g. singleApproverWorkflow = "Active";
2. Not using the macro for the account creation and just execute the steps one by one since we don't actually need to set anything but the basic information:
   e.g. 
    
 ```
                 Type(
			locator1 = "TextInput#NAME",
			value1 = "${discountName}");

		Select(
			key_fieldLabel = "Type",
			locator1 = "Select#GENERIC_SELECT_FIELD",
			value1 = "${discountType}");

		Select(
			key_fieldLabel = "Apply To",
			locator1 = "Select#GENERIC_SELECT_FIELD",
			value1 = "${discountTarget}");

		          Button.click(button = "Submit");

                        AssertTextEquals(locator1= "<statusLocator>", value1 = "DRAFT")

                      Click(
				key_text = "Submit for Publication",
				locator1 = "Button#ANY");
           
                     AssertTextEquals(locator1= "<statusLocator>", value1 = "PENDING (CREATED)")
```

NOTE: This is an example, if you think that any other function fits better, use it.